roles/setup_pgd/tasks/pg_hba_config.yml

```diff
@@ -57,6 +57,64 @@
   run_once: true
   no_log: "{{ disable_logging }}"
 
+- name: Prepare hba non ssl list for user - pg_owner
+  ansible.builtin.set_fact:
+    pg_allow_ip_addresses: >-
+      {{ pg_allow_ip_addresses | default([]) + [
+        {
+          "contype": "host",
+          "users": pg_owner,
+          "source": node.private_ip + "/32",
+          "databases": "postgres"
```

Comment:
Can we use a parameter for the `databases` value? This only allows for the database to be named `postgres`

Reply:
Yeah, I believe this should be it's own parameter.  We don't necessarily want the default database (pg_database?) also be the initial replicated database. Having said that, I don't recall whether the initial replicated database is hard coded...